site2/docs/schema-get-started.md

```diff
@@ -11,80 +11,283 @@ import TabItem from '@theme/TabItem';
 ````
 
 
-This hands-on tutorial provides instructions and examples on how to construct and customize schemas.
+This hands-on tutorial provides instructions and examples on how to construct schemas. For the instructions on administrative tasks, see [Manage schema](admin-api-schemas.md).
 
-## Construct a string schema
+## Construct a schema
 
-This example demonstrates how to construct a [string schema](schema-understand.md#primitive-type) and use it to produce and consume messages in Java.
+- [Construct a bytes schema](#bytes)
+- [Construct a string schema](#string)
+- [Construct a key/value schema](#keyvalue)
+- [Construct an Avro schema](#avro)
+- [Construct a JSON schema](#json)
+- [Construct a Protobuf schema](#protobuf)
+- [Construct a ProtobufNative schema](#protobufnative)
+- [Construct a native Avro schema](#nativeavro)
+- [Construct an AUTO_PRODUCE schema](#auto_produce)
+- [Construct an AUTO_CONSUME schema](#auto_consume)
 
-1. Create a producer with a string schema and send messages.
+### `bytes`
+
+This example demonstrates how to construct a [bytes schema](schema-understand.md#primitive-type) using language-specific clients and use it to produce and consume messages.
+
+````mdx-code-block
+<Tabs groupId="api-choice"
+  defaultValue="Java"
+  values={[{"label":"Java","value":"Java"},{"label":"C++","value":"C++"},{"label":"Python","value":"Python"},{"label":"Go","value":"Go"}]}>
+
+<TabItem value="Java">
+
+```java
+       .topic("my-topic")
+       .create();
+Consumer<byte[]> consumer = pulsarClient.newConsumer(Schema.BYTES)
```

Comment:
Seems missing some code here

Reply:
@liangyepianzhou can you pls revisit this code snippet?

Reply:
```java
        Producer<byte[]> producer = client.newProducer(Schema.BYTES)
```
I think adding this part would be ok.